_episodes_rmd/13-dplyr.Rmd

```diff
@@ -263,6 +263,35 @@ gdp_pop_bycontinents_byyear <- gapminder %>%
               sd_gdp_billion=sd(gdp_billion))
 ```
 
+### Connect mutate with logical filtering: ifelse
+
+When creating new variables, we can hook this with a logical condition. A simple combination of 
+`mutate()` and `ifelse()` facilitates filtering right where it is needed: in the moment of creating something new.
+This easy-to-read statement is a fast and powerful way of discarding certain data (even though the overall dimension
+of the data.frame will not change) or for updating values depending on this given conditon.
```

Comment:
The style we settled on for talking about data frames is to say "data frame" without the quotes when we are referring to the object, and `data.frame()` when referring to the function.